x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -15,6 +15,7 @@ class org.elasticsearch.xpack.sql.expression.function.scalar.whitelist.InternalS
   boolean nullSafeFilter(Boolean)
   double nullSafeSortNumeric(Number)
   String nullSafeSortString(Object)
+  Number nanSafeFilter(Number)
```

Comment:
This name is misleading - the above methods are used to wrap an expression used in filtering; this method should be used against all aggs, regardless of their position or whether the expression is a filter or not.
How about `nullAgg()`?